webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const webpack = require('webpack')
+const path = require('path')
+
+module.exports = (env, options) => ({
+	mode: 'development',
```

Comment:
Ah yes, if you run `npm run build` it will default to `development`, but to run a production build you have to specify `npm run build --mode=production`.
Feel free to change this to whatever behavior you would prefer :)

Reply:
Thanks for explaining. I'll approve it the way it is and then change those behaviours.
Another thing, will you push the .gitignore file or do you prefer that I do it myself?